todo-app/src/app/api/add-todo/route.ts

```diff
@@ -0,0 +1,20 @@
+import { NextRequest, NextResponse } from "next/server";
+import db from "../../../../db/connection";
+import { todoSchema } from "../../../../db/schema";
+
+export async function POST(request: NextRequest) {
+  const { title: title, description: description } = await request.json();
+
+  try {
+    await db
+      .insert(todoSchema)
+      .values({ title: title, description: description });
+    return NextResponse.json({ status: 200 });
+  } catch (error) {
+    console.error(error);
+    return NextResponse.json({
+      status: 500,
```

Comment:
you should never return 500 status code, as this essentially means, that your server crashed.